packages/react-components/src/form/src/FormContext.ts

```diff
@@ -1,9 +1,14 @@
 import { createContext, useContext } from "react";
 import { isNil } from "lodash";
 
-export const FormContext = createContext(null);
+export interface FormContextProps {
```

Comment:
If you use this name for the actual Context instance how would you call an interface for `useFormContext` if it eventually receive args?

Reply:
Well I guess you could call it "UseFormProps".

That being said, suffixing a type describing an actual context with "props" seems wrong.

I would have call it "FormContext" but we obviously can't.

Reply:
i'll do this in the next PR